src/interfaces/restoration-interfaces.ts

```diff
@@ -5,9 +5,9 @@ import { RestorationTypes } from '@/enums'
  * https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/restoration.json
  */
 export interface RestorationIF {
-  date: string // FUTURE: describe format here
   type: RestorationTypes
   expiry?: string // FUTURE: describe format here
+  legalName?: string
```

Comment:
What is this for? I don't think it's in the schema.

Reply:
That's true, it's not in the schema, but the business has requested the legalName as at the time of restoration.  Let's discuss this in a call.

Reply:
OK. Need to chat with Vysakh as well (as he worked on the schema most recently).